### PR TITLE
Adjust pixel comparison tolerance for a test to pass on certain hardw…

### PR DESCRIPTION
**Pull request by zhenyao**
…are.

### CODE REVIEW

File: sdk/tests/conformance2/reading/read-pixels-from-fbo-test.html [hunk 1]
```diff
@@ -144,9 +144,13 @@
...
     case gl.RGBA4:
     case gl.SRGB8_ALPHA8:
-    case gl.RGB10_A2:
       srcIsNormalized = true;
       tol = 5;
+    case gl.RGB10_A2:
+      srcIsNormalized = true;
+      // tol == 33 is needed on Linux/NVIDIA for the test to pass.
+      // Might need to further loosen the tol on other GPUs/drivers.
+      tol = 40;
```
Comment (kenrussell) [inline sdk/tests/conformance2/reading/read-pixels-from-fbo-test.html]: That's quite a large tolerance -- could that possibly indicate a bug in the test? If you've verified this looks basically correct, then LGTM.

**Comment by kenrussell** [comment]
LGTM with a question. Please merge yourself after answering. Thanks.

File: sdk/tests/conformance2/reading/read-pixels-from-fbo-test.html (hunk 1 above)
Comment (zhenyao) [inline sdk/tests/conformance2/reading/read-pixels-from-fbo-test.html]: Good question.  Let me check the normalizing code.

**Comment by zhenyao** [comment]
Updated. Please take another look.

**Comment by kenrussell** [comment]
Thanks for double-checking. LGTM. Merging now.

**Pull request merged**